packages/web/src/scripts/util.js

```diff
@@ -166,13 +167,13 @@ export function generateAAP(fileType, editorInstance) {
  * @param { Monaco Object } editorInstance - Monaco editor object
  */
 export function generateAAS(fileType, editorInstance) {
-    const assetInterfaceDescriptionUtil = new AssetInterfaceDescriptionUtil()
+    const assetInterfaceDescription = new AssetInterfacesDescription()
 
     const tdToConvert = fileType === "json"
         ? editorInstance.getValue()
         : convertTDYamlToJson(editorInstance.getValue())
 
-    const AASInstance = assetInterfaceDescriptionUtil.transformTD2SM(tdToConvert)
+    const AASInstance = assetInterfaceDescription.transformTD2AID(tdToConvert, { createAAS: false }, [])
```

Comment:
```suggestion
    const AASInstance = assetInterfaceDescription.transformTD2AID(tdToConvert)
```

You should be able to simplify it.